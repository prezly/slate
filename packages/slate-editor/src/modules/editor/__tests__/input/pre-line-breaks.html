<html>
    <head>
        <meta charset="utf-8" />
    </head>
    <body>
        <pre
            class="ndfHFb-c4YZDc-fmcmS-DARUcf"
            style="
                user-select: text;
                display: block;
                font-family: 'Courier New', Courier, monospace, arial, sans-serif;
                margin: 0px;
                white-space: pre-wrap;
                overflow-wrap: break-word;
                background-color: rgb(255, 255, 255);
                color: rgb(0, 0, 0);
                font-size: 14px;
                font-style: normal;
                font-variant-ligatures: normal;
                font-variant-caps: normal;
                font-weight: 400;
                letter-spacing: normal;
                orphans: 2;
                text-align: left;
                text-indent: 0px;
                text-transform: none;
                widows: 2;
                word-spacing: 0px;
                -webkit-text-stroke-width: 0px;
                text-decoration-thickness: initial;
                text-decoration-style: initial;
                text-decoration-color: initial;
            "
        >
this is a title&#10;&#13;paragraph immediately below, just one cr/lf inserted&#10;&#13;asdasdasdas&#10;&#13;asdasdasdas&#10;&#13;asdasdasd</pre
        >
    </body>
</html>
