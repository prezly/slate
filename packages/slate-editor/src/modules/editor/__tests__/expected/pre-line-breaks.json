[
    { "type": "paragraph", "children": [{ "text": "this is a title" }] },
    { "type": "paragraph", "children": [{ "text": "" }] },
    {
        "type": "paragraph",
        "children": [{ "text": "paragraph immediately below, just one cr/lf inserted" }]
    },
    { "type": "paragraph", "children": [{ "text": "" }] },
    { "type": "paragraph", "children": [{ "text": "asdasdasdas" }] },
    { "type": "paragraph", "children": [{ "text": "" }] },
    { "type": "paragraph", "children": [{ "text": "asdasdasdas" }] },
    { "type": "paragraph", "children": [{ "text": "" }] },
    { "type": "paragraph", "children": [{ "text": "asdasdasd" }] },
    { "children": [{ "text": "" }], "type": "paragraph" }
]
